packages/dependency-extraction-webpack-plugin/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.2.0
+
+-   Add WooCommerce Blocks Dependencies.
```

Comment:
Given you referenced the PR change for this in the comments #6228 do we want to add that to the changelog here?

Reply:
Good call! Done in https://github.com/woocommerce/woocommerce-admin/pull/6268/commits/e5c001d6ae7f50bb715ce9b61c81c4d14461047b